### PR TITLE
feat: add sevennet support to mlp recipes

### PR DESCRIPTION
**Pull request by mamunm**
## Summary of Changes
Added **sevennet** to the MLP recipes based on issue #2429. 

Key changes include:

1. Added sevennet implementation to mlp recipes
2. Created unit tests to validate sevennet functionality
3. Ensured compatibility with existing MLP workflows

## Requirements

-[X] My PR is focused on a single feature addition [#2429 ].
-[X] My PR has relevant, comprehensive unit tests.
-[X] My PR is on a custom branch (feature/sevennetMLP).

### CODE REVIEW

**Comment by buildbot-princeton** [comment]
Can one of the admins verify this patch?

File: src/quacc/recipes/mlp/_base.py [hunk 1]
```diff
@@ -64,6 +65,16 @@ def pick_calculator(
             kwargs["default_dtype"] = "float64"
         calc = mace_mp(**kwargs)
 
+    elif method.lower() == "sevennet":
+        from sevenn import __version__
+        from sevenn.sevennet_calculator import SevenNetCalculator
+
+        if "model" not in kwargs:
+            kwargs["model"] = "7net-0"
```
Comment (Andrew-S-Rosen) [inline src/quacc/recipes/mlp/_base.py:73]: 7net-0 is the default model, so it seems like we should probably not specify this.

File: src/quacc/recipes/mlp/_base.py [hunk 2]
```diff
@@ -64,6 +65,16 @@ def pick_calculator(
...
+        if "device" not in kwargs:
+            kwargs["device"] = "cpu"
```
Comment (Andrew-S-Rosen) [inline src/quacc/recipes/mlp/_base.py:75]: The default device is "auto". Is there a reason to override this default by forcing CPU?

**Review by Andrew-S-Rosen — COMMENTED** [review]
@mamunm: Thank you very much for your PR! I have just two questions before I can merge this.

File: src/quacc/recipes/mlp/_base.py (hunk 1 above)
Comment (mamunm) [reply src/quacc/recipes/mlp/_base.py:73]: I added both the default settings so the user knows what model and devices are being used if they want to dig deeper. But it's redundant, and I'll remove them in the next push.

Comment (Andrew-S-Rosen) [reply src/quacc/recipes/mlp/_base.py]: The default setting is "auto" for device I believe, but in any case let's remove them so we are always consistent with the underlying calculator.

**Comment by Andrew-S-Rosen** [comment]
Thanks, merging!

**Comment by mamunm** [comment]
> Thanks, merging!

Thanks for the review!

**Pull request merged**